2017/OCT/CHEFGP/CHEFGP.py

```diff
@@ -0,0 +1,63 @@
+t = int(input())
+while t > 0:
+    s = input()
+    w = ""
+    x,y = map(int, input().split())
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpw1suy8dc/2017/OCT/CHEFGP/CHEFGP.py
+++ b/tmp/tmpw1suy8dc/2017/OCT/CHEFGP/CHEFGP.py
@@ -2,7 +2,7 @@
 while t > 0:
     s = input()
     w = ""
-    x,y = map(int, input().split())
+    x, y = map(int, input().split())
     na = s.count("a")
     nb = s.count("b")
     if na > nb:
```